lib/bokken/birthday_notifier.ex

```diff
@@ -0,0 +1,26 @@
+defmodule Bokken.BirthdayNotifier do
+  @moduledoc """
+  Executes a task periodically to notify Ninjas on their birthday.
+  """
+
+  alias Bokken.Accounts
+  alias BokkenWeb.EventsEmails
+  import Bokken.Events.EventAdmin
+
+  def send_happy_birthday_ninjas do
+    ninjas = Accounts.list_ninjas([:user])
+    current_time = Date.utc_today()
+
+    ninjas
+    |> Enum.filter(fn ninja ->
+      ninja.birthday.month == current_time.month && ninja.birthday.day == current_time.day &&
+        not is_nil(ninja.user)
+    end)
+    |> Enum.map(fn ninja -> ninja.user end)
+    |> send_email(fn user ->
+      EventsEmails.ninja_birthday_email(user.ninja,
+        to: Accounts.get_guardian_email_by_ninja(ninja: user)
+      )
```

Comment:
The situation is as follows. Currently not a single ninja has a registered account, meaning this is basically useless. What we could do is send the email to the guardian if the ninja does not have an account associated with them

Reply:
But Ninjas are not even supposed to have emails, when I'm using the .email I thought I was sending it to the guardians right?